Suggest (#376)
canonical/sphinx-docs-starter-pack: docs/reference/style-guide.rst

  :discourse: 12345

> Man-page links
> ^^^^^^^^^^^^^^

Reviewer: ```suggestion
Manual-page links
^^^^^^^^^^^^^^^^^
```

When mentioning command line utilities, you may wish to link to the

> corresponding man-page for the command. Ensure that the ``manpages_url``

Reviewer: ```suggestion
corresponding manual page for the command. Ensure that the ``manpages_url``
```

setting in your :file:`conf.py` is set appropriately and use the ``:manpage:``
inline role within your text to create a link.

> For example, to link to man-pages from the 24.04 LTS (Noble Numbat) release,

Reviewer: (The webz seem to like the non-hyphenated version more when not an adjective.)
```suggestion
For example, to link to man pages from the 24.04 LTS (Noble Numbat) release,
```

> ensure the following is present in your :file:`conf.py`:

Reviewer: (Just to get rid off the second 'ensure'.)
```suggestion
include the following in your :file:`conf.py`:
```

.. code-block:: python

    manpages_url = "https://manpages.ubuntu.com/manpages/noble/en/man{section}/{page}.{section}.html"

> Then within your documentation you can use the following reST:

Reviewer: ```suggestion
Then within your documentation, use the following reST:
```

- For local pictures, start the path with :file:`/` (for example, :file:`/images/image.png`).
- Use ``PNG`` format for screenshots and ``SVG`` format for graphics.
- If producing multiple output formats, use ``*`` as the file extension to have

>   sphinx select the best image format for the output

Reviewer: ```suggestion
  Sphinx select the best image format for the output
```
